src/app/zap-templates/zcl/zcl.json

```diff
@@ -133,6 +133,37 @@
             "TxErrCount",
             "CollisionCount",
             "OverrunCount"
+        ],
+        "General Diagnostics": [
+            "NetworkInterfaces",
```

Comment:
Fwiw, putting list-typed things in here does not hurt anything, but is not necessary either: lists are detected by type and always forced to be external anyway.

Reply:
Thanks!